client/src/components/Feed/PostSocial.js

```diff
@@ -198,6 +204,56 @@ const PostSocial = ({
           <StyledSpan>Share</StyledSpan>
         </div>
       </div>
+
+      <span></span>
+      {postId ? (
+        // NOTE: May need new function to trigger message sending feature on click
+        <div id={gtmTag("message", GTM.post.prefix)} className="social-icon">
```

Comment:
Please update these to the right comment formatting.

Reply:
Thank you for pointing these out!